packages/libs/react-ui/src/components/Form/Select/Select.test.tsx

```diff
@@ -59,7 +59,7 @@ describe('Select', () => {
     await userEvent.click(selectContainer);
     await userEvent.click(screen.getByRole('option', { name: 'Option 2' }));
     expect(handleChange).toHaveBeenCalledTimes(1);
-  });
+  }, 20000);
```

Comment:
it just throws timeout on CI without increasing the timeout

Reply:
This is fine, the test is also slow locally. We might have to think about increasing the size of our GitHub runners